ingressPort.adoc

```diff
@@ -659,7 +659,7 @@ Load successful; *ldata* valid: Error. Load failed; *ldata* not valid
 |*lid*[_lrid_width_p_-1:0] | S | Split Load ID. Valid when *lresp* is
 non-zero
 |*sdata*[_sdata_width_p_-1:0] | S | Store data. Valid when *dretire* is
-high
+high and access is a store or atomic.
```

Comment:
Should it say "atomic memory operation" to avoid any ambiguity.

Reply:
How about "if access isa  store (dtype = 1) or atomic (dtype = 8 - 14)"?

Reply:
That would be good.

Reply:
Done :-)